js/demo.js

```diff
@@ -0,0 +1,23 @@
+
+
+
+
+
+ function square() {
+   let input = prompt('Please enter a  number');
+   let x = Number(input);
+   let result = x *  x ;
+
+   alert(result);
+ }
+
+
+function multiplier() {
+  let input1 = prompt("Please enter a number");
+  let x = Number(input1);
+  let input2 = prompt("Please enter another number");
```

Comment:
Nice work! You can also chain these functions together:
`let x = Number(prompt("Please enter a number"));`